ScriptHookVDotNet.sln

```diff
@@ -2,38 +2,105 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
 VisualStudioVersion = 14.0.25420.1
-MinimumVisualStudioVersion = 12.0.40629.0
+MinimumVisualStudioVersion = 12.0.25420.1
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Loader", "Loader.vcxproj", "{B2933D8F-F922-40BD-BB70-18622A81AB8F}"
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ScriptHookVDotNet", "ScriptHookVDotNet.vcxproj", "{019193A7-50C0-444A-84CA-777595E702CD}"
-EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ScriptHookVDotNetExamples", "ScriptHookVDotNetExamples.csproj", "{A717AD5D-C5B5-4769-BD40-F14C09F269BB}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ScriptHookVDotNet", "ScriptHookVDotNet.csproj", "{61165C1A-6BFE-4776-B371-5A3FDB1A2ED7}"
 	ProjectSection(ProjectDependencies) = postProject
-		{019193A7-50C0-444A-84CA-777595E702CD} = {019193A7-50C0-444A-84CA-777595E702CD}
+		{5D98F169-2298-4ECD-977E-24619104AF5D} = {5D98F169-2298-4ECD-977E-24619104AF5D}
 	EndProjectSection
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ScriptHookVWrapperDotNet", "ScriptHookVWrapperDotNet.vcxproj", "{5D98F169-2298-4ECD-977E-24619104AF5D}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
+		NativeGen|Any CPU = NativeGen|Any CPU
```

Comment:
On that note: The `NativeGen` target is no longer implemented. That should be fixed. I'm not sure how well it works in a csproj MSBuild file, but we could always move it to the "Loader" project if it doesn't.